Portal/src/Datahub.Portal/Pages/Public/RegisterPage.razor

```diff
@@ -93,7 +93,7 @@
         <NotAuthorized>
             <MudAlert Severity="Severity.Error" Dense="true" Class="my-2">
                 <MudText Typo="Typo.body1">
-                    @Localizer["The production environment is not available yet. Use the proof-of-concept environment instead."] 
+                    @Localizer["The Federal Science DataHub is launching this March. To preview the platform, please register in the proof of concept environment"] 
```

Comment:
This needs rewording - we need to disable self-registration in PoC

Reply:
something like `The Federal Science DataHub is launching this March. To preview the platform, please contact us to access the proof of concept environment`